src/WebJobs.Script.WebHost/Security/Authentication/Jwt/ScriptJwtBearerExtensions.cs

```diff
@@ -110,15 +111,26 @@ private static IEnumerable<string> GetValidAudiences()
             string runtimeSiteName = ScriptSettingsManager.Instance.GetSetting(AzureWebsiteRuntimeSiteName);
             var audiences = new List<string>
             {
-                string.Format(SiteAzureFunctionsUriFormat, siteName),
-                string.Format(SiteUriFormat, siteName)
+                string.Format(SiteUriFormat, siteName),
+                string.Format(SiteAzureFunctionsUriFormat, siteName)
```

Comment:
The change here is just an optimization, since the SiteUriFormat is the 99% case so should be ordered first.